erpnext/hr/doctype/employee_grievance/employee_grievance.js

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) 2021, Frappe Technologies Pvt. Ltd. and contributors
+// For license information, please see license.txt
+
+frappe.ui.form.on('Employee Grievance', {
+	setup: function(frm) {
+		frm.set_query('grievance_against_party', function() {
+			return {
+				filters: {
+					name: ['in', [
+						'Company', 'Department', 'Employee Group', 'Employee Grade', 'Employee']
+					]
+				}
+			};
+		});
+		frm.set_query('associated_document_type', function() {
+			return {
+				filters: {
+					name: ['in', [
+						"Salary Slip",
+						"Payroll",
+						"Employee Benefits",
+						"Expense Claim",
+						"Leave Encashment",
+						"Employee Incentive"
+					]]
+				}
+			};
+		});
+	},
+	refresh: function(frm) {
+		if (!frm.doc.__islocal && frm.doc.status === "Resolved" && frm.doc.docstatus === 1) {
+			if (frm.doc.is_applicable_for_suspension && (!(frm.doc.suspended_from && frm.doc.suspended_to) && !frm.doc.unsuspended_on)) {
+				frm.add_custom_button(__("Suspend Employee"), function () {
+					frm.events.suspend_or_unsuspend_employee(frm, 'suspend');
+				});
+			}
+
+			if (frm.doc.is_applicable_for_pay_cut) {
+				frm.add_custom_button(__("Apply Pay-cut"), function () {
+					frm.events.create_additional_salary(frm, );
+				});
+			}
+
+			if (frm.doc.suspended_from && frm.doc.suspended_to) {
+				let suspended_from = frappe.datetime.global_date_format(frm.doc.suspended_from);
+				let suspended_to = frappe.datetime.global_date_format(frm.doc.suspended_to);
+
+				let message_line1 = "Employee: <b>"+ frm.doc.employee_responsible +"</b> is suspended from <b>"+ suspended_from+"</b> to <b>" +suspended_to + "</b>.";
+				let message_line2 = "Employee will be unsuspended automatically or you can do it manually by clicking on unsuspend Employee.";
```

Comment:
Not translation-friendly. This message is too long.

```suggestion
				let message = __("Employee {0} is suspended from {1} to {2}. {0} will be unsuspended automatically on {2}.", [frm.doc.employee_responsible, suspended_from, suspended_to]);
```
The button label is clear enough to indicate that the user can manually unsuspend the employee. Do we need to mention it explicitly?

Reply:
done